onmt/modules/multi_headed_attn.py

```diff
@@ -206,8 +260,7 @@ def forward(self, key: Tensor, value: Tensor,
             query = self.linear_query(query)
             key = shape(key, self.dim_per_head)
             value = shape(value, self.dim_per_head)
-
-        query = shape(query, self.dim_per_head)
+            query = shape(query, self.dim_per_head)
```

Comment:
Specific reason for moving this in the condition?

Reply:
yes because we need to shape query just before applying rotary above.